.github/pull_request_template.md

```diff
@@ -2,22 +2,35 @@
 
 Please include a summary of the fix/feature/change, including any relevant motivation and context.
 
-Link to any related issue(s): 
+Link to any related issue(s):
 
 ## Type of change:
 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
+  expected)
 - [ ] This change requires a documentation update
 
+## Manual QA performed:
+
+- [ ] cfn invoke for each of CRUDL/cfn test
+- [ ] Updated resource in ../example
+- [ ] Published to AWS private registry
+- [ ] Created and updated resource in AWS
```

Comment:
```suggestion
- [ ] Use the template in [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples) to create a stack in AWS
```

Reply:
@andreaangiolillo do we not want to test updating a resource?

Reply:
yes, I missed it in my suggested changes